inst/rmarkdown/templates/pdf/skeleton/pinp.cls

```diff
@@ -206,14 +206,7 @@
 \RequirePackage[noend]{algpseudocode}
 \RequirePackage{changepage}
 \RequirePackage[twoside,%
-				includeheadfoot,
-                left=38.5pt,%
-                right=43pt,%
-                top=43pt,% 10pt provided by headsep
-                bottom=32pt,%
-                headheight=0pt,% No Header
-                headsep=10pt,%
-                footskip=25pt]{geometry}
+				includeheadfoot]{geometry}
```

Comment:
Part of the charm of pinp is adherence to the PNAS style.

Reply:
I understand and I agree as I didn't like the idea of getting rid of the PNAS defined settings for letter paper. Anyway before I discuss further I will open a separate issue ticket.